code/src/index.css

```diff
@@ -11,3 +11,83 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.main-container {
+  font-size: 13px;
+  padding-top: 100px;
+  width: 400px;
+  margin: auto;
+}
+
+.form-container {
+  height: 200px;
+  background: #f2f0f0;
+  border: 1px solid;
+  box-shadow: 7px 7px black;
+}
+
+.input-form {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 80%;
+  padding: 20px;
+}
+
+.form-button {
+  border-radius: 50px;
+  border: none;
+  height: 30px;
+  width: 50%;
+  background: #ffadad;
+  font-size: 11px;
+}
+
+.message-container {
+  height: 150px;
+  border: 1px solid;
+  margin-top: 30px;
+  box-shadow: 7px 7px black;
+}
+
+.message-content {
+  padding: 15px 20px;
+}
+
+.message-text {
+  word-wrap: break-word;
+  height: 65px;
+}
+
+.message-details {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.hearts {
+  padding: 10px;
+  background: #eaeaea;
+  text-align: center;
+  margin-right: 10px;
+  border-radius: 50%;
+  width: 20px;
+  height: 20px;
+}
+
+.hearts-count {
+  margin-right: auto;
+}
+
+.char-counter {
+  text-align: end;
+}
+
+.message-time {
+  opacity: 0.5;
+  font-size: 11px;
+}
+
+.heart {
+  font-size: 13px;
+}
```

Comment:
Not much to say about the styling, it looks good to me, good indentations and readable code 💯